slides/index-en.html

```diff
@@ -0,0 +1,1304 @@
+              <li>3.4. Add the class <code>show</code> to it.</li>
+            </ul>
+          </li>
+        </ul>
+        <i class=" fa fa-angle-right"></i>
+      </section>
+      <section>
+        <p class="note question">Are you unsure about any of the instructions? Now would be the good time to spend some more time with them to make sure you know what is happening.</p> <i class=" fa fa-angle-right"></i>
+      </section>
+      <section>
+        <p class="note task">Using what we have learnt already, add the <code>showNextSlide</code> function in the <code>gallery.js</code> file and add hiding the currently being displayed element in it. Assing this function
+          to clicking in the right arrow.
+        </p>
+        <i class=" fa fa-angle-right"></i>
+      </section>
+      <section>
+        <p>An example of the solution:</p>
+        <pre><code class="lang-javascript">function showNextSlide( ) {
+   document.querySelector( '.show' ).classList.remove( 'show' );
+   // the rest of our code will go here
+}
+document.querySelector( '#next' ).addEventListener( 'click', showNextSlide );</code></pre>
+        <p>Let's see how it works!</p>
+        <i class=" fa fa-angle-right"></i>
+      </section>
+      <section>
+        <p class="note question">The challenge here is finding the next element to the one that was clicked.
+          Do you have any ideas how to achieve that?</p>
+        <i class=" fa fa-angle-right"></i>
+      </section>
+      <section>
+        <p>We need to remember, which slide was viewed most recently and then calculate which would be the next one.</p>
+        <p>Now, to achieve that, we will use <strong>variables</strong>.</p>
+        <i class=" fa fa-angle-right"></i>
+      </section>
+      <section>
+        <p class="note more">A <strong>variable</strong> is a storage place where we can store information
+          and then read them in a different place. In JavaScript variables can store anything - 
+          text, number, document element, complex object or even a function.</p>
+        <i class=" fa fa-angle-right"></i>
+      </section>
+      <section>
+        <p>To create <strong>a variable</strong> in JavaScript we use the key word <code>let</code>:</p>
+        <pre><code class="lang-javascript">//text variable
+let courseTitle = 'JavaScript gallery';</code></pre>
+        <p>Once created the variable we can use it anywhere else in the code with just using its name:</p>
+        <pre><code class="lang-javascript">//number variable - the text ,ength
+let titleLength = courseTitle.length;</code></pre>
+        <i class=" fa fa-angle-right"></i>
+      </section>
+      <section>
+        <p class="note question">As mentioned before, we can store anything in a variable. Let's have a moment to think,
+          what format of data we should use to store the information of which slide is currently being viewed, so then
+          we can easily count which would be the nex one.</p>
```

Comment:
```suggestion
        <p class="note question">As mentioned before, we can store anything in a variable. Let's think for a moment: 
          what format of data should we use to store the information about the currently viewed slide, so that
          we can easily count which will be the next one?</p>
```